Adding X and Reddit to Bitgree card in ecosystem.html

josuegfernandez opened the pull request.

minisat0shi reviewed (changes requested):
Very minor -- please remove the "www." and the trailing "/" from the reddit URL then I can merge these in. Thanks!

josuegfernandez commented:
I've added a commit to remove the www and the trailing slash. 👍

Note: Reddit will actually do two 301 redirects though, one to add the www and one to add the trailing slash. That's why I added the www and the trailing slash to the link, to save the user those two redirects. 😇

minisat0shi commented:
> I've added a commit to remove the www and the trailing slash. 👍
> 
> Note: Reddit will actually do two 301 redirects though, one to add the www and one to add the trailing slash. That's why I added the www and the trailing slash to the link, to save the user those two redirects. 😇

Interesting -- did not know this. I do it to save the couple extra bytes, especially on such a large page. But perhaps I'll put www. and trailing slash for all reddit links then.. will be quick, at least for the www.'s. Thanks for the heads up!

minisat0shi reviewed (commented):
:D
